compiler/src/parsing/driver.re

```diff
@@ -167,10 +167,14 @@ let read_imports = ({Parsetree.comments, Parsetree.statements}) => {
       },
     );
   let found_includes = ref([]);
-  let iter_mod = (self, import) =>
-    found_includes := [import.Parsetree.pinc_path, ...found_includes^];
-  let iterator = {...Ast_iterator.default_iterator, include_: iter_mod};
-  List.iter(iterator.toplevel(iterator), statements);
+
+  let enter_include = inc => {
+    found_includes := [inc.Parsetree.pinc_path, ...found_includes^];
+  };
+
+  let include_iter = Parsetree_iter.(make({...default_hooks, enter_include}));
+
+  List.iter(include_iter.iter_toplevel_stmt, statements);
```

Comment:
This sort of always irked me; could we change this to `iter_parsed_program`?

Reply:
Yep! I thought about doing that too but wasn't sure

Reply:
You missed this one!